### PR TITLE
Wrapper for primerclip.

### PR DESCRIPTION
**Pull request by Smeds**

### CODE REVIEW

File: bio/primerclip/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+rule primerclip:
+    input:
+        master_file="master_primer_file",
+        sam="mapped/{sample}.sam"
```
Comment (johanneskoester) [inline bio/primerclip/test/Snakefile]: Does it work with bam or cram as well? If not, the wrapper should probably automatically add samtools piping. Nobody should be forced to use sam if not needed ;-).

Comment (Smeds) [reply bio/primerclip/test/Snakefile]: Good suggestion! I'm able to add samtool conversion for the input. But I get a truncation error when I try to do it for the output, primerclip outputs a file with sam format.

Comment (johanneskoester) [reply bio/primerclip/test/Snakefile]: But can't you pipe the output into `samtools view -Sbh`?

Comment (Smeds) [reply bio/primerclip/test/Snakefile]: Manage to find out why I couldn't pipe the output successfully. Primerclip outputs some run information to stdout which mess up the alignment file. I solved it by removing the 3 last lines from the primerclip output, which contained that information.

Comment (Smeds) [reply bio/primerclip/test/Snakefile]: Fixed the formatting issue a few minutes ago.

Comment (Smeds) [reply bio/primerclip/test/Snakefile]: Updated Snakemake file

**Comment by Smeds** [comment]
Don't really understand why SonarCloud fail=/

**Pull request merged**